### PR TITLE
Fix parsing of minimized css output (2)

### PR DESCRIPTION
**Pull request by rubenstolk**
https://github.com/Polymer/polymer/pull/2689 but with fixed tests

### CODE REVIEW

**Comment by rubenstolk** [comment]
@azakus @TimvdLippe I think this one is good to be merged now

**Comment by rubenstolk** [comment]
@googlebot have a look at https://github.com/Polymer/polymer/pull/2689 and add `cla: yes` :)

**Comment by rubenstolk** [comment]
@TimvdLippe do you think we should make the same change to `mixinProp`?

**Comment by TimvdLippe** [comment]
Hm I vaguely remember I also applied the fix to `mixinProp` but I think it broke some tests. It seems logical imo and worth another shot.

**Comment by rubenstolk** [comment]
I've applied the same fix to `mixinProp` as well now in this PR. @TimvdLippe would you mind cross-checking the code and then assigning it to @azakus to merge it?

File: test/unit/custom-style.html [hunk 1]
```diff
@@ -245,13 +248,15 @@
     <template>
       <style>
         child-of-child-with-var {
+          position: relative;
```
Comment (TimvdLippe) [inline test/unit/custom-style.html]: We might want to add a CSS comment to clarify the usage of this rule.

**Comment by TimvdLippe** [comment]
One small comment, otherwise LGTM! Thanks for your investigation and fix!

**Comment by rubenstolk** [comment]
Added a comment! @TimvdLippe please assign it to @azakus so we can get this merged!

**Comment by TimvdLippe** [comment]
I don't have write access to this repository, so I can not assign anyone.

**Comment by rubenstolk** [comment]
@TimvdLippe just noticed you're in Delft too. Time for a Polymer@Delft meetup!?

**Comment by TimvdLippe** [comment]
@rubenstolk contacted you on the Polymer Slack

**Comment by azakus** [comment]
LGTM!

**Comment by rubenstolk** [comment]
Yay @azakus, now when https://github.com/PolymerLabs/polyclean/pull/9 gets merged we finally have a real solution for minifying Polymer!!!

**Pull request merged**